src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -1,28 +1,28 @@
 from pyqtgraph.Qt.QtCore import Qt
 from pyqtgraph.Qt.QtWidgets import *
 import time
+import software.python_bindings as tbots
 
 from software.thunderscope.thread_safe_buffer import ThreadSafeBuffer
 from software.thunderscope.common import common_widgets
 from proto.import_all_protos import *
 
-# TODO (#2683) get these values from robot constants
-MAX_DRIBBLER_RPM = 10000
-MIN_DRIBBLER_RPM = -10000
+constants = tbots.create2021RobotConstants()
 
-MAX_LINEAR_SPEED_MPS = 5
-MIN_LINEAR_SPEED_MPS = -5
+MAX_DRIBBLER_RPM = constants.max_force_dribbler_speed_rpm
+MIN_DRIBBLER_RPM = -MAX_DRIBBLER_RPM
 
-MAX_ANGULAR_SPEED_RAD_PER_S = 20
-MIN_ANGULAR_SPEED_RAD_PER_S = -20
+MAX_LINEAR_SPEED_MPS = constants.robot_max_speed_m_per_s
+MIN_LINEAR_SPEED_MPS = -MAX_LINEAR_SPEED_MPS
+
+MAX_ANGULAR_SPEED_RAD_PER_S = constants.robot_max_ang_speed_rad_per_s
+MIN_ANGULAR_SPEED_RAD_PER_S = -MAX_ANGULAR_SPEED_RAD_PER_S
```

Comment:
nit: we shouldn't have function calls outside of classes to avoid it running when we import this file.

Reply:
I think we can reduce the variable count by removing the all caps variables, instead, we can initialize `constants = tbots.create2021RobotConstants()` as a member variable inside the class.
i.e.
```python
class DriveAndDribblerWidget(QWidget):
    def __init__(self, proto_unix_io):
        ....
        self.constants = tbots.create2021RobotConstants()
```
this way, where ever we need a constant, we can just get it from the variable. So instead of 
```python
common_widgets.create_slider(
            "X (m/s)", MIN_LINEAR_SPEED_MPS * 1000, MAX_LINEAR_SPEED_MPS * 1000, 1
        )
```
we can just get the value directly
```python
common_widgets.create_slider(
            "X (m/s)", -self.constants.robot_max_speed_m_per_s * 1000, self.constants.robot_max_speed_m_per_s * 1000, 1
        )
```
Let me know what you think